cmd/banner.go

```diff
@@ -90,13 +90,14 @@ var bannerCmd = &cobra.Command{
 					}
 				}
 				if i == 0 {
-					fmt.Println("Waiting for Supervisor to startup...")
+					fmt.Println("Waiting for Supervisor to start-up...")
 				}
 				time.Sleep(1 * time.Second)
 			}
+			fmt.Println("Supervisor is ready!\n\n")
 		}
 
-		fmt.Println("System information")
+		fmt.Println("System Information:")
```

Comment:
We typically use Microsoft style guide for headers, so this should remain lower caps.

```suggestion
		fmt.Println("System information:")
```